index.js

```diff
@@ -96,12 +96,15 @@ export default class SlimSlider{
    */
   initDom(){
     if(!this.parent.querySelector('.slim-slides')){
+      let slidesWrapper = create('div', {class: 'slim-slides-wrapper'});
       this.slider = create('div', {class:'slim-slides'})
       this.slides.forEach( slide => {
         this.slider.appendChild(slide)
-      })
-      this.parent.appendChild(this.slider)
+      });
+      slidesWrapper.appendChild(this.slider);
```

Comment:
in case of slidesWrapper is already there in the dom? what will happen? 
that's why I encourage you to add to `this.slideWrapper`

because you're declaring it inside the if condition.